go-app-ussd_public_rapidpro.js

```diff
@@ -411,17 +411,17 @@ go.app = function() {
             return new ChoiceState(name, {
                 // TODO: Proper copy
                 question: $(
-                    "We may occasionally send messages for historical, ... " +
-                    "We'll keep her info safe. Does she agree?"
+                    "We may occasionally call or send msgs for historical/statistical/research reasons. " +
+                    "We’ll keep your info safe. Do you agree?"
                 ),
                 error: $(
-                    "Sorry, please reply with the number next to your answer. We may occasionally send msgs for " +
-                    "... Do you agree?"
+                    "Sorry, please reply with the number next to your answer. We may call or send " + 
+                    "msgs for reaserch reasons. Do you agree?"
```

Comment:
There seems to be a typo here on `reaserch`

Reply:
thanks for picking this up